student.rb

```diff
@@ -0,0 +1,11 @@
+# Student Class Inheriting From Person Class
+class Student < Person
+  def initialize(classroom)
+    super
+    @classroom = classroom
+  end
```

Comment:
- [ ] This method should have the arguments from the parent, and "classroom" added to it.  Super should initialize the parent method with its required arguments like this:

```ruby
def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
end
```

That way, the class will be properly initialized! 💪

Reply:
@Meltrust, Thank you so much for your review :star: